### ISSUE TITLE
Repos skipped updating

### ISSUE THREAD

**Issue by epage** [epage/_rust#20]
Wanting to track what I'm skipping and need to get back to
- clap
- cobalt
- committed

**Comment by epage** [comment]
Problems with wildcard imports
- git-fixture
- git-config-env
- git-conventional
- anstyle

**Comment by epage** [comment]
False positive in escargot
```rust
#![allow(clippy::self_named_module_files)] // false positive
```
